storage/imx6ull-flash/imx6ull-flash-test.c

```diff
@@ -326,12 +395,16 @@ void test_ecc(void)
 	const unsigned int rawdatasz = 512 + 23;                                  /* 512B DATA + 22.75B ECC14 (adding 2 bits for byte alignment) */
 	const unsigned int maxmetaecc = 16;                                       /* Max number of bits per meta chunk ECC can correct */
 	const unsigned int maxdataecc = 14;                                       /* Max number of bits per data chunk ECC can correct */
```

Comment:
Why not camelCase? `maxmetaecc` is not readable.

Reply:
Not a part of this PR. To be fixed in another one.